src/main/java/homework1/Main.java

```diff
@@ -0,0 +1,13 @@
+package homework1;
+
+public class Main {
+
+    public static String wordAndNumber (String word){
+
+        if (word.equals("ошибка")) {
+           return "\u001b[31m" + " ТРЕВОГА";
```

Comment:
You forgot closing escape string. Also brackets after else. Also I don't see public static void main anywhere